make/wag.mk

```diff
@@ -1,23 +1,34 @@
 # This is the default Clever Wag Makefile.
 # Please do not alter this file directly.
-WAG_MK_VERSION := 0.1.0
+WAG_MK_VERSION := 0.2.0
 SYSTEM := $(shell uname -a | cut -d" " -f1 | tr '[:upper:]' '[:lower:]')
+WAG_INSTALLED = $(shell [[ -e "bin/wag" ]] && bin/wag --version)
+WAG_LATEST = $(shell curl -S https://api.github.com/repos/Clever/wag/releases/latest 2>/dev/null | grep tag_name | cut -d\" -f4)
 
 SHELL := /bin/bash
-.PHONY: wag-update-makefile wag-generate-deps
+.PHONY: bin/wag wag-update-makefile wag-generate-deps ensure_wag_version_set
 
-./wag:
-	curl -L https://github.com/Clever/wag/releases/download/v$(WAG_VERSION)/wag-$(WAG_VERSION)-$(SYSTEM)-amd64.tar.gz | tar -xz
+ensure_wag_version_set:
+	@ if [ "${WAG_VERSION}" = "" ]; then \
```

Comment:
Also, I suggest using double brackets (`[[ "$(WAG_VERSION)" = "" ]]`). Not super relevant here but the `/bin/sh`/POSIX version of conditionals (single brackets) is in general a world of hurt compared to the bash version (double brackets).